tools/pythonpkg/pyduckdb/spark/tests/basic/test_dataframe_read.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+class TestSparkSession(object):
+	def test_read_csv_basic(self, spark):
+		pass
```

Comment:
Is this intentionally left blank?

Reply:
It's not implemented as part of this PR yea, the whole reading/writing is big enough to warrant a separate PR I think, I'll remove the "tests" for it